google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/standard/AppEngineTemplateGroupDescriptorFactory.java

```diff
@@ -32,12 +33,19 @@
 
   @NonNls
   public static final String APP_ENGINE_WEB_XML_TEMPLATE = "GctAppEngineWeb.xml";
+  @NonNls
+  public static final String APP_YAML_TEMPLATE = "app.yaml";
```

Comment:
yea, my next planned PR for this (templatizing the remaining instances of app.yaml / Dockerfile) may address this